docker/nginx/conf.d/dhparam.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN DH PARAMETERS-----
```

Comment:
should this be public?

Reply:
yeah this is ok to be public https://ssl-config.mozilla.org/ffdhe2048.txt